mwm-5-dynamic-delivery/presentation/PITCHME.md

```diff
@@ -468,6 +468,20 @@ SplitInstallManager.deferredUninstall(List<String> moduleNames)
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Load native library</span><span style="text-transform: none; font-size:0.8em;"> in dynamic module</span>
+
+```kotlin
+System.loadLibrary("my-native-lib-name")
+```
+
+Won't work for a library inside dynamic module. Instead use:
+
+```kotlin
+SplitInstallHelper.loadLibrary(context, "my-native-lib-name")
```

Comment:
Some comment as the one of Vincent, maybe replace `"my-native-lib-name"` by `"search-feature-native-lib"` (if we want to be consistent with the "search" dynamic feature that we have bellow. (could be something else, no pb)

Reply:
fixed